includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1576,10 +1576,12 @@ public function wc_setup_activate_save() {
 			exit;
 		}
 
-		$redirect_url   = site_url( add_query_arg( array(
+		$redirect_url   = add_query_arg( array(
```

Comment:
Does this need to be escaped?

Reply:
`get_admin_url` is filterable by plugins. It's not high-risk at all, but implementing `esc_url_raw` would not be a bad idea IMO.